pkg/apis/postgres-operator.crunchydata.com/v1beta1/patroni_types.go

```diff
@@ -23,6 +23,10 @@ type PatroniSpec struct {
 	// +kubebuilder:validation:Minimum=3
 	LeaderLeaseDurationSeconds *int32 `json:"leaderLeaseDurationSeconds,omitempty"`
 
+	// Patroni log configuration settings.
+	// +optional
+	Log *PatroniLogConfig `json:"log,omitempty"`
```

Comment:
🤔 This matches the Patroni naming, but I want to deviate from Patroni exposes. What do you think of naming this `logging`?

```yaml
spec:
  patroni:
    logging:
      level: INFO
```